.github/workflows/test-all.yml

```diff
@@ -0,0 +1,74 @@
+on: [pull_request]
+
+name: cargo test
+
+jobs:
+  test:
+    name: Test Suite
+    runs-on: ubuntu-latest
+    container: ubuntu:latest
+
+    services:
+      postgres:
+        image: postgres
+        env:
+          POSTGRES_PASSWORD: password
+          POSTGRES_DB: auth_service_db
```

Comment:
Shouldn't matter since it's only for testing but maybe put those in as [GitHub Secret](https://docs.github.com/en/actions/reference/encrypted-secrets#using-encrypted-secrets-in-a-workflow)?